script.js

```diff
@@ -1,4 +1,226 @@
+//live server
+//index of to find a position number of an input
+
+//dice roll
+//main function
+//number of players --> store into var
+//number of dices --> store into var
+//name of players --> while statement --> store into array
+//player 1 roll --> while statement number of dices --> store into array
+//player 1 decision --> store into array
+//player 2 roll etc --> if --> number of players = player number --> go next
+//show all player rolls in order
+//scoreboard --> mathmax --> index of --> parallel array --> array[index]
+//new game
+
+var gameMode = "number of dice rolls";
```

Comment:
global variables and helper functions makes things lot simpler and cleaner! well done